components/DateTimePicker.js

```diff
@@ -1,73 +1,73 @@
-import React, {Component} from 'react'
-import {Text, View, TouchableOpacity, Modal} from 'react-native'
+import React, { Component } from 'react'
```

Comment:
This file's content isn't changed except the spacing. R u able to revert the change to this file?